tests/catalog/test_base.py

```diff
@@ -255,6 +258,10 @@ def catalog() -> InMemoryCatalog:
     return InMemoryCatalog("test.in.memory.catalog", **{"test.key": "test.value"})
 
 
+TEST_PYARROW_SCHEMA = pa.schema([
```

Comment:
nit: i think this is no longer used, tests are using `pyarrow_schema_simple_without_ids` from conftest instead

Reply:
You are right! Thank you for the nit